src/flags.c

```diff
@@ -2,3 +2,21 @@
 bool FlagParseComments;
 bool FlagShowErrorCodes;
 bool FlagVerbose = true;
+
+void parseFlag(const char *flagName, bool *out, int argc, char **argv) {
+    for (int i = 1; i < argc; i++) {
```

Comment:
Add an `ASSERT(out)` here to guard against somebody calling this will a `out` as `NULL`